src/components/button/button.vue

```diff
@@ -10,86 +10,103 @@ module.exports = {
     label: String,
     modifier: String,
     disabled: Boolean
+  },
+  computed: {
+    modifierClass() {
+      return this.modifier ? 'apos-button--' + this.modifier : false;
+    }
+  },
+  methods: {
+    click() {
+    	this.$emit('click');
+    }
   }
 }
 </script>
 
 <style lang="scss">
   .apos-button {
-    color: var(--neutral-eight);
-    background: var(--primary);
-    padding: 10px 20px; // re use this?
+    color: var(--text-primary);
+    background-color: var(--default-button);
+    padding: 10px 20px;
     border-radius: var(--border-radius);
     border: none;
-    font-size: 1.3em;
+    font-size: map-get($font-sizes, modal);
```

Comment:
Are we using em? I thought rem was generally speaking the most recommended/human-readable. I have type styles organized this way in Figma...not sure if this lines up with your thinking. A base size and small size, with 3 variations in letter-spacing (0, .5, 1). Body Copy and Heading for modals and Table titles etc.  A few utilities.

<img width="236" alt="Screen Shot 2020-03-11 at 2 09 50 PM" src="https://user-images.githubusercontent.com/2650558/76464192-1cf6a780-63a2-11ea-88c6-6e4c0e4f9033.png">